build/yaml/skillFunctionalTestsBuild.yml

```diff
@@ -0,0 +1,40 @@
+steps:
+# Note: Template ci-build-steps.yml is not supported in macOS because it calls VSBuild@1 in order to build the Windows-only ASP.NET Desktop assemblies.
```

Comment:
This comment doesn't really apply here, consider removing it

Reply:
removed